src/main/java/org/lantern/LanternUtils.java

```diff
@@ -1371,4 +1371,14 @@ public static boolean isShuttingDown() {
         }
         return false;
     }
+
+    /**
+     * Utility for whether or not we're running on a Pi. Just checks for ARM 
+     * for now.
+     * 
+     * @return <code>true</code> if we're on ARM, otherwise <code>false</code>.
+     */
+    public static boolean isLanternPi() {
+        return SystemUtils.OS_ARCH.toLowerCase().contains("arm");
```

Comment:
If you make this a command-line flag and use the OS_ARCH check to set the default of that flag, then we can test the lantern-pi code path on our usual development machines.